test/lint/lint-python-mutable-default-parameters.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+Detect when a mutable list or dict is used as a default parameter value in a Python function.
+"""
+
+import subprocess
+import sys
+
+
+def main():
+    command = "git grep -E".split(" ")
```

Comment:
`command = ["git", "grep", "-E". ]`, no need to call split here

Reply:
Done, I put the whole command into an array